controllers/util.go

```diff
@@ -159,18 +159,12 @@ func ensureDeletePipelineIsReconciled(jobOpts jobOpts) (ctrl.Result, error) {
 	}
 
 	if existingDeletePipeline == nil {
+		jobOpts.logger.Info("Creating Delete Pipeline. The pipeline will now execute...")
+
+		//TODO retrieve error information from applyResources to return to the caller
 		applyResources(jobOpts.opts, jobOpts.pipelineResources...)
+
 		return defaultRequeue, nil
-		// jobOpts.logger.Info("Creating Delete Pipeline. The pipeline will now execute...")
-		// //TODO come back and address- do we know for sure that the cluster roles and service accounts will exist?
```

Comment:
These todos (the one you removed vs the one you added) are pretty different in concept. Is there a reason we have moved from needing to verify the roles vs needing to retrieve error info?

Reply:
Ah yep - this TODO was addressed by the changes in #67 (we now ensure the roles/service accounts exist by creating them in the pipeline resources in `delete.go`)!